<launch>
	<node pkg="joy" type="joy_node" name="joy_node" >
		<param name="dev" value="/dev/input/by-id/usb-Logitech_Logitech_Cordless_RumblePad_2-joystick" />
		<param name="autorepeat_rate" value="10.0" />
	</node>

	<param name="/CAN_device" value="/dev/usb/cpc_usb0" />
	<node pkg="nifti_ros_drivers" type="nifti_robot_node"
		name="nifti_robot_node" output="screen" />

	<node pkg="nifti_teleop" type="nifti_teleop_joy.py"
		name="nifti_teleop_joy" output="screen" />
</launch>
